### ISSUE TITLE
Missing notebooks in docs?

### ISSUE THREAD

**Issue by donovanr** [vanvalenlab/deepcell-tf#301]
I'm interested in checking out some example notebooks here
https://deepcell.readthedocs.io/en/documentation/readme_link.html#getting-started

for example 
https://deepcell.readthedocs.io/en/documentation/scripts/deepcell/DeepCell%20Transfrom%203D%20Sample%20Based.ipynb

but I'm getting
``` 
        \          SORRY            /
         \                         /
          \    This page does     /
           ]   not exist yet.    [ 
```

for all of them.  Should they exist?  Is there anywhere I can see examples of segmenting cell nuclei in 3d?

**Comment by willgraf** [comment]
Sorry about that, our docs are currently under construction!  The notebooks probably should not link to readthedocs, and instead can be found here: [https://github.com/vanvalenlab/deepcell-tf/tree/master/scripts](https://github.com/vanvalenlab/deepcell-tf/tree/master/scripts)

**Comment by donovanr** [comment]
ok, thanks!

**Comment by willgraf** [comment]
Fixed by #320
